__tests__/app.test.js

```diff
@@ -82,8 +82,70 @@ describe("get api", () => {
       .get("/api")
       .expect(200)
       .then((response) => {
-        const { apiDetails } = response.body;
-        expect(apiDetails).toEqual(endpoints);
+        const { apiData } = response.body;
+        expect(apiData).toEqual(endpoints);
+      });
+  });
+});
+
+describe("get articles by id", () => {
+  test("should respond with an object with correct keys ", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(typeof article).toBe("object");
+        expect(Array.isArray(article)).toBe(false);
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.article_id).toBe("number");
+        expect(typeof article.body).toBe("string");
+        expect(typeof article.topic).toBe("string");
+        expect(typeof article.created_at).toBe("string");
+        expect(typeof article.votes).toBe("number");
+        expect(typeof article.article_img_url).toBe("string");
+      });
+  });
+
+  test("should return correct data for article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        const expected = {
+          article_id: 1,
```

Comment:
I can see you checked it here, however I think this could have been asserted in the one test above.